Initial project build and attempt to load in Android Studio fails

When cloning repo then opening the project for the first time in Android Studio (`Android Studio Narwhal | 2025.1.1 Patch 1` -- detailed Android Studio version info at bottom), I'm getting an error in the IDE:

<img width="2055" height="1283" alt="Image" src="https://github.com/user-attachments/assets/2ba048db-9a62-448d-b6e4-8707c6475e12" />

When issuing build command from read me, there are also errors:

```
$ ./gradlew installDebug
Type-safe project accessors is an incubating feature.
Project accessors enabled, but root project name not explicitly set for 'sensortag'. Checking out the project in different folders will impact the generated code and implicitly the buildscript classpath, breaking caching.

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:packageDebug'.
> Could not create task ':app:compileDebugJavaWithJavac'.
   > Failed to calculate the value of task ':app:compileDebugJavaWithJavac' property 'javaCompiler'.
      > Cannot find a Java installation on your machine (Mac OS X 15.5 aarch64) matching: {languageVersion=17, vendor=any vendor, implementation=vendor-specific, nativeImageCapable=false}. Toolchain download repositories have not been configured.

* Try:
> Learn more about toolchain auto-detection and auto-provisioning at https://docs.gradle.org/8.14.3/userguide/toolchains.html#sec:auto_detection.
> Learn more about toolchain repositories at https://docs.gradle.org/8.14.3/userguide/toolchains.html#sub:download_repositories.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 2s

```

Detailed Android Studio version info:
```
Android Studio Narwhal | 2025.1.1 Patch 1
Build #AI-251.25410.109.2511.13752376, built on July 8, 2025
Runtime version: 21.0.6+-13391695-b895.109 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.lwawt.macosx.LWCToolkit
macOS 15.5
Kotlin plugin: K2 mode
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 4096M
Cores: 10
Metal Rendering is ON
Registry:
  ide.experimental.ui=true
  com.android.studio.ml.activeModel=com.android.studio.ml.AidaModel
  gradle.phased.sync.enabled=true
Non-Bundled Plugins:
  com.intellij.marketplace (251.25410.159)
  org.jetbrains.completion.full.line (251.25410.999.1)
  com.jetbrains.kmm (0.9-251.25410-AS-69)
  com.anthropic.code.plugin (0.1.11-beta)
  org.jetbrains.junie (251.167.97)
  org.norbye.tor.kdocformatter (1.6.8)
```

--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -7,6 +7,12 @@ pluginManagement {
     }
 }
 
+plugins {
+    // Provides repositories for auto-downloading JVM toolchains.
+    // https://github.com/gradle/foojay-toolchains
+    id("org.gradle.toolchains.foojay-resolver-convention") version "1.0.0"
+}
+
 include(
     "app",
     "bluetooth",

